lib/vehicle.js

```diff
@@ -5,6 +5,16 @@ const _ = require('lodash');
 
 /** @exports Vehicle */
 
+/**
+ * @param {Object} response
+ *
+ * @return {Date|null} A parsed age or null if no age exists
+ */
+const parseAge = function(response) {
```

Comment:
Noticed in the tests that we get the age as a string. But our avro returns ms. Does this conversion happen somewhere else?

Reply:
That conversion happens in our API. You can see the format of our `SC-Data-Age` [here
](https://smartcar.com/docs#response-headers)